### DISCUSSION TITLE
编解码功能合在一起

### DISCUSSION THREAD

**Discussion by ddgszc** [he3-app/he3-feedback#24] [Ideas]
比如Base64编解码，Hex转ASCII等编解码，可以放在一个页面内，不然编码解码还需要点开两个页面

**Comment by zhenweiwang1990** [comment]
感谢建议，后续我们会逐步的把更多工具的编码解码这种双向互转的场景做到一起。

**Comment by hongchaodeng** [comment]
已修复 https://github.com/he3-app/he3-feedback/issues/28
